cms/models/pluginmodel.py

```diff
@@ -321,7 +321,7 @@ def update(self, refresh=False, **fields):
         return
 
     def reload(self):
-        return CMSPlugin.objects.get(pk=self.pk)
+        return CMSPlugin.objects.select_related("parent", "placeholder", "placeholder").get(pk=self.pk)
```

Comment:
@fsbraun Is the second `placeholder` inside the `select_related` redundant?

Reply:
Yes, it is. 🤦‍♂️

Reply:
```suggestion
        return CMSPlugin.objects.select_related("parent", "placeholder").get(pk=self.pk)
```